homeassistant/components/mealie/strings.json

```diff
@@ -23,7 +33,16 @@
           "host": "[%key:common::config_flow::data::url%]",
           "api_token": "[%key:common::config_flow::data::api_token%]",
           "verify_ssl": "[%key:common::config_flow::data::verify_ssl%]"
+        },
+        "data_description": {
+          "host": "[%key:component::mealie::common::data_description_host%]",
+          "api_token": "[%key:component::mealie::common::data_description_api_token%]",
+          "verify_ssl": "[%key:component::mealie::common::data_description_verify_ssl%]"
         }
+      },
+      "hassio_confirm": {
+        "title": "Mealie via Home Assistant add-on",
+        "description": "Do you want to configure Home Assistant to connect to the Mealie instance provided by the add-on: {addon}?"
       }
```

Comment:
We don't have this one yet

Reply:
Oops, guess which branch I was on 😄 
Removed!